README.md

```diff
@@ -181,8 +184,8 @@ DummyElasticSearchModel.where(_q: { my_string: "Hi", my_int: 5..10) })
 # Query by attributes where at least 1 value matches
 DummyElasticSearchModel.where(_q: { my_string: "Hi", my_int: [1, (5..10)] })
 
-# Query by nested classes or hash fields
-DummyElasticSearchModel.where(_q: { my_nested_class: { nested_hash_field: { a: 1, b: 2 } } })
+# Query by nested class fields (hashes and nested hashes are not queryable by default)
+DummyElasticSearchModel.where(_q: { my_nested_class: { nested_string_field: "Hey" })
```

Comment:
This definitely worked before...so why are we making it so that we can't do this now?

Reply:
Since the default for hashes is becoming JSON blobs, the query won't find a `my_nested_class.nested_hash_field.a` field anymore.